graylog2-plugin-interfaces/src/main/java/org/graylog2/plugin/buffers/MessageEvent.java

```diff
@@ -45,6 +46,7 @@ public MessageEvent newInstance()
 
     private RawMessage raw;
     private Message msg;
+    private List<Message> msgList;
```

Comment:
Could we make this field a `Collection<Message>` and just call it `messages`? I don't think there's any advantage in using a `List` here.